overlay.d/05rhcos/usr/libexec/rhcos-fail-boot-for-older-luks-syntax

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -euo pipefail
+
+# This script tries to look for a `/etc/clevis.json` reference 
+# (used to enable LUKS in RHCOS 4.6 or earlier versions) in an 
+# Ignition config. If it exists and the version of RHCOS is 4.7, 
+# then the script ends with exit 1.
+
+ignition_cfg="/run/ignition.json"
+wanted_path="/etc/clevis.json"
+tmpconfig=${outputdir}/tmpconfig.json
+
+source /etc/os-release
+if [ "$VERSION_ID" == "4.7" ]; then
```

Comment:
We still want to emit this error in e.g. 4.8 right?  I think if we move this to be properly in the initramfs and also if the unit does `ConditionKernelCommandLine=ignition.firstboot` we don't need a version check at all, because the unit will only run on the first boot.